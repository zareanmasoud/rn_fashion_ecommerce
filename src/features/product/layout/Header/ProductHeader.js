import React from 'react';
import MainNavbar from 'features/layout/MainNavbar';
// import styles from './styles';

export default function ProductHeader() {
  return <MainNavbar cartTextTestID="product_cart_text" />;
}
