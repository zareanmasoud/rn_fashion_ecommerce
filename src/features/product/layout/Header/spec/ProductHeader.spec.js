import React from 'react';
import {shallow} from 'enzyme';
import ProductHeader from '../ProductHeader';

describe('ProductHeader Feature', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const wrapper = shallow(<ProductHeader />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
