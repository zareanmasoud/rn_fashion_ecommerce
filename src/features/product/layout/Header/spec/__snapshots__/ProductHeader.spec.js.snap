// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductHeader Feature Rendering should match to snapshot 1`] = `
<withNavigation(MainNavbar)
  cartTextTestID="product_cart_text"
/>
`;
